chunky/src/java/se/llbit/math/Octree.java

```diff
@@ -67,7 +68,7 @@
     int getData(NodeId node);
     default void startFinalization() {}
     default void endFinalization() {}
-    default Pair<NodeId, Integer> getWithLevel(int x, int y, int z) {
+    default void getWithLevel(IntIntMutablePair typeAndLevel, int x, int y, int z) {
```

Comment:
Maybe we should rename the first argument to `out` to make clear that it is used to return a value (or add javadoc)?